playbooks/ci-ceph_swift/tasks/pre-deploy.yml

```diff
@@ -35,6 +35,32 @@
     - name: umount ephemeral disk on ceph_osds
       shell: umount /dev/{{ item }}
       with_items: "{{ ceph.disks }}"
+      ignore_errors: true
+
+- name: umount ephemeral disk on swiftnode
+  hosts: swiftnode
+  tasks:
+    - name: umount ephemeral disk on swiftnode
+      shell: umount /dev/{{ item.disk }}
+      with_items: "{{ swift.disks }}"
+      ignore_errors: true
+
+- name: create new partition table for ephemeral disk on swiftnode
+  hosts: swiftnode
+  tasks:
+    - name: create new msdos partition table for ephemeral disk
+      shell: parted /dev/{{ item.disk }} --script mklabel msdos
+      with_items: "{{ swift.disks }}"
+
+- name: deploy fake lshw file on swiftnode
+  hosts: swiftnode
+  tasks:
+    - name: rename the original /usr/bin/lshw
+      command: mv /usr/bin/lshw /usr/bin/lshw.original
+    - name: deploy fake lshw file /usr/bin/lshw
+      template: src=../templates/usr/bin/lshw
+                dest=/usr/bin/lshw
+                owner=root group=root mode=0755
```

Comment:
Anyway, we should always do this update-alternatives stuff in our pre-deploy.yml, it can't be included in the standard list of update-alternatives for swift roles in ursula code since it is valid for our test in VMs only.